bundle/deploy/terraform/convert.go

```diff
@@ -279,6 +290,11 @@ func TerraformToBundle(state *resourcesState, config *config.Root) error {
 			src.ModifiedStatus = resources.ModifiedStatusCreated
 		}
 	}
+	for _, src := range config.Resources.SecretScopes {
+		if src.ModifiedStatus == "" {
```

Comment:
Other resources (except for apps) also check that ID is empty string. If you don't check it, the resource is always incorrectly marked as "created" even though it is not.